### PR TITLE
added dev docker-compose file

### PR DESCRIPTION
**Pull request by xanderle**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,3 +32,17 @@ npm run lint
...
 npm run dev
 ```
+
+## Docker
+
+For ease of use, there is a docker-compose file container MongoDB and the NodeJS server.
+follow the same configuration steps then run
+
+```sh
+docker-compoose up
```
Comment (w3cj) [inline README.md:42]: Typo! docker-compoose -> docker-compose

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+services:
+  api:
+    build: .
+    ports:
+      - 3000:3000
+    depends_on:
+      - mongodb
+    environment:
+      - PORT=3000
```
Comment (w3cj) [inline docker-compose.yml:11]: There are quite a few environment variables the container will need. Can you edit this so it reads in from the `.env` file? You can see all the variables in the `.env.sample` file.

See here for an example of pulling in a value from the `.env`: https://github.com/CodingGarden/entropychat.app/blob/master/server/docker-compose.yml#L28

**Review by w3cj — CHANGES_REQUESTED** [review]
Thanks for doing these! See my comments above.

File: docker-compose.yml (hunk 2 above)
Comment (xanderle) [reply docker-compose.yml:11]: what about using
```
env_file:
      - .env
``` 
just for the sake of verbosity

**Pull request closed without merging**